eng/CredScanSuppression.json

```diff
@@ -4,41 +4,40 @@
         {
             "placeholder": [
                 "secret=",
```

Comment:
Do we have plans on removing this as well?

Reply:
I am assuming this is not the final PR of cleaning up. The placeholder like this is still risky.